idl/github.com/uber/cadence/shared.thrift

```diff
@@ -911,6 +911,7 @@ struct History {
 
 struct WorkflowExecutionFilter {
   10: optional string workflowId
+  20: optional string runId
```

Comment:
@vancexu is it correct to add this to this struct?

Reply:
I have talked to @vancexu about this before. He's ok with this change.

Reply:
I think it is ok to have RunID as optional filter.